cms/management/commands/startcmsproject.py

```diff
@@ -0,0 +1,105 @@
+
+from cms import __version__ as cms_version
+
+
+class Command(TemplateCommand):
+    help = (
+        "Creates a django CMS project directory structure for the given project "
+        "name in the current directory or optionally in the given directory."
+    )
+    missing_args_message = "You must provide a project name."
+
+    def get_default_template(self):
+        from cms import __version__
+
+        version = ".".join(__version__.split(".")[:2])  # get major.minor
+        return f"https://github.com/django-cms/cms-template/archive/{version}.tar.gz"
+
+    def postprocess(self, project):
+        self.HEADING = self.style.SQL_FIELD
+        self.install_requirements(project)
+        os.chdir(project)
+        self.stdout.write(self.HEADING("Create migrations"))
+        self.run_management_command(["migrate"], capture_output=True)
+        self.stdout.write(self.HEADING("Create superuser"))
+        self.run_management_command(["createsuperuser"])
+        self.stdout.write(self.HEADING("Check installation"))
+        self.run_management_command(["cms", "check"])
+
+        message = f"django CMS {cms_version} installed successfully"
+        separator = "=" * len(message)
+        self.stdout.write(self.HEADING(f"{separator}\n{message}\n{separator}"))
+        self.stdout.write(f"""
+Congratulations! You have successfully installed django CMS,
+the lean enterprise content management powered by Django!
+
+Now, to start the development server first go to your newly
+created project:
+$ {self.style.SUCCESS("cd " + project)}
+$ {self.style.SUCCESS("./manage.py runserver")}
+
+Learn more at https://docs.django-cms.org/
+Join the django CMS Slack channel http://www.django-cms.org/slack
+
+Enjoy!
+""")
+
+    def install_requirements(self, project):
+        for req_file in ("requirements.txt", "requirements.in"):
+            requirements = os.path.join(project, req_file)
+            if os.path.isfile(requirements):
+                if self.running_in_venv():
+                    self.stdout.write(self.HEADING(f"Installing requirements in {requirements}"))
+                    result = subprocess.run(
+                        [sys.executable, "-m", "pip", "install", "-r", requirements],
+                        capture_output=True,
+                    )
+                    if result.returncode:
+                        self.stderr.write(result.stderr)
+                        raise CommandError(f"Failed to install requirements in {requirements}")
+                    break
+                else:
+                    self.stdout(self.style.ERROR("To automatically install requirements for your new django CMS "
+                                "project use this command in an virtual environment."))
+                    raise CommandError("Requirements not installed")
+
+    def run_management_command(self, commands, capture_output=False):
+        result = subprocess.run(
+            [sys.executable, "./manage.py"] + commands,
```

Comment:
I'd rather this did;

```suggestion
            [sys.executable, "-m", "manage"] + commands,
```

Reply:
I think it needs to be `python manage.py ...`